frontend/src/App.vue

```diff
@@ -34,15 +34,15 @@ export default {
       return this.$auth.user()
     },
     deploymentTime () {
-      const timestamp = window.environment.DEPLOYMENT_TIME
+      const timestamp = window.environment?.DEPLOYMENT_TIME
```

Comment:
should the environment be nullable?

Reply:
Leftover from my deployment to netlify before we had kubernetes deployment. Reverted with https://github.com/ecamp/ecamp3/pull/2480/commits/ef356d5a0c1005d7fd1341aacedb888856f59bea